src/lib/build.mjs

```diff
@@ -45,6 +45,8 @@ export const getBuildOptions = ({
   cwd,
   featureFlags: {
     ...edgeFunctionsFeatureFlags,
+    // eslint-disable-next-line unicorn/no-useless-fallback-in-spread
+    ...(cachedConfig.siteInfo.feature_flags ?? {}),
```

Comment:
I'm curious: why are you adding the fallback and disabling the ESLint rule when the fallback is unnecessary?

```js
> {foo: "bar", ...null, ...undefined}
{ foo: 'bar' }
>
```